src/components/Button/Button.css

```diff
@@ -1,21 +1,21 @@
 .button {
   border-radius: 25px;
-  background: #6600FF;
-  border: 1px solid #6600FF;
-  color: #FFFFFF;
+  background: #6600ff;
+  border: 1px solid #6600ff;
+  color: #ffffff;
   outline: none;
   border: none;
   font-size: 16px;
   line-height: 19px;
   padding: 16px 45px;
   cursor: pointer;
   font-family: 'renner_mediummedium', Helvetica, sans-serif;
-  transition: .2s all;
+  transition: 0.2s all;
```

Comment:
just as a quick check, have you looked into the docs on what `all` does? It means that all css properties transition. It is also possible to select which specific properties you want to transition, and that can be a lot more performant and precise. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions

Reply:
yes